SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec/mapped_v1_dtdlv2_Brick_1_3-REC_4_0.json

```diff
@@ -0,0 +1,420 @@
+{
+  "Header": {
+    "InputOntologies": [
+      {
+        "Name": "mapped",
+        "Version": "1.0",
+        "DtdlVersion": "v2"
+      }
+    ],
+    "OutputOntologies": [
+      {
+        "Name": "brick",
+        "Version": "1.3",
+        "DtdlVersion": "v2"
+      },
+      {
+        "Name": "rec",
+        "Version": "4.0",
+        "DtdlVersion": "v2"
+      }
+    ]
+  },
+  "NamespaceRemaps": [
+    {
+      "InputNamespace": "dtmi:mapped:core:",
+      "OutputNamespace": "dtmi:org:brickschema:schema:Brick:"
+    }
+  ],
+  "InterfaceRemaps": [
+    {
+      "InputDtmi": "dtmi:mapped:core:Ablutions_Room;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:ShowerRoom;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Access_Control_Zone;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:AccessControlZone;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Address;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:PostalAddress;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Atrium;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:Atrium;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Auditorium;1",
+      "OutputDtmi": "dtmi:org:w3id:rec:Auditorium;1"
+    },
```

Comment:
Should there be a NamespaceRemaps for mapped and rec?

Reply:
Can currently only map an incoming namespace to a single outgoing namespace. Everything else is an exception.